app/src/main/res/layout/fragment_walkthrough_welcome.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context=".walkthrough.welcome.WalkthroughWelcomeFragment">
+  <!-- TODO: Update blank fragment layout -->
+  <TextView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:text="@string/hello_blank_fragment" />
+</FrameLayout>
```

Comment:
Please add End of File. i.e. one empty line at end of file.

Reply:
done